src/docs/index.md

```diff
@@ -0,0 +1,151 @@
+#title: "Sample mapping set"
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification
+HP:0008551	Microtia	skos:exactMatch	MP:0000018	small ears	semapv:LexicalMatching
+HP:0000411	Protruding ear	skos:exactMatch	MP:0000021	prominent ears	semapv:LexicalMatching
+```
+
+### Quick links
+
+**General**
+
+- [GitHub page](https://github.com/mapping-commons/sssom)
+- [Detailed description](introduction.md)
+- [Formal specification](spec-intro.md)
+
+**Publications**
+
+- [A Simple Standard for Sharing Ontological Mappings (SSSOM)](https://doi.org/10.1093/database/baac035) (initial publication in _Database_)
+- [A Simple Standard for Ontological Mappings 2022: Updates of data model and outlook](https://zenodo.org/record/7672104) (paper and presentation at the Ontology Matching Workshop 2022)
+- [A Simple Standard for Ontological Mappings 2023: Updates on data model, collaborations and tooling](https://zenodo.org/record/8202395) (paper and presentation at the Ontology Matching Workshop 2023)
+- [Other presentations](presentations.md)
+
+**Related software**
+
+- [SSSOM Toolkit](https://mapping-commons.github.io/sssom-py/) (reference implementation of the standard, in Python)
+
+## The SSSOM Core Team
+
+### Contact
+
+The preferred way to contact the SSSOM team is through the [issue tracker](https://github.com/mapping-commons/issues) (for problems with SSSOM) or the [GitHub discussion forums](https://github.com/mapping-commons/sssom/discussions) (for general questions).
+
+You can find any of the members of the SSSOM core team [on GitHub](https://github.com/orgs/mapping-commons/teams/sssom-core). Their GitHub profiles usually also provide email addresses.
+
+You can also reach us in the [OBO Foundry Slack](https://obo-communitygroup.slack.com/archives/C01DP18L5GW), in the `#sssom` channel.
+
+### Documentation/specification editors
+
+* [Nicolas Matentzoglu](https://orcid.org/0000-0002-7356-1779) (Semanticly Ltd; [@matentzn](https://github.com/matentzn))
+* [Chris Mungall](https://orcid.org/0000-0002-6601-2165) (LBL)
+* [Ernesto Jimenez-Ruiz](https://orcid.org/0000-0002-9083-4599) (City, University of London)
+* [John Graybeal](https://orcid.org/0000-0001-6875-5360) (Stanford)
+* [William Duncan](https://orcid.org/0000-0001-9625-1899) (LBL)
+* [David Osumi-Sutherland](https://orcid.org/0000-0002-7073-9172) (EMBL-EBI)
+* [Simon Jupp](https://orcid.org/0000-0002-0643-3144) (SciBite)
+* [James McLaughlin](https://orcid.org/0000-0002-8361-2795) (EMBL-EBI)
+* [Henriette Harmse](https://orcid.org/0000-0001-7251-9504) (EMBL-EBI)
+* [Tiffany Callahan](https://orcid.org/0000-0002-8169-9049) ([@callahantiff](https://github.com/callahantiff))
+* [Charlie Hoyt](https://orcid.org/0000-0003-4423-4370) (Harvard Medical School; [@cthoyt](https://github.com/cthoyt))
+* [Thomas Liener](https://orcid.org/0000-0003-3257-9937) (Pistoia Alliance)
+* [Harshad Hegde](https://orcid.org/0000-0002-2411-565X) (LBL)
```

Comment:
Hum, no _strong_ objection, but I don’t like having _anything_ in the documentation that would be merely a pointer to GitHub. The documentation should be, as much as possible, “standalone”.

In fact, if the list of contributors is merely a pointer to a GitHub team, this creates a problem, because if a contributor leaves the team at so point, then he or she will no longer appear in the documentation, as if they had never been a contributor in the first place. I believe this is wrong. If someone has contributed to the doc/the spec, their name should stay in the list even after they leave the team. 

~~(Unless they leave the team to work on a competing project, in which case their name should fall into oblivion for eternity.)~~

Reply:
Of note, if we do keep the list here (and update it of course):

I just realised that the ORCID organisation is one of those that have the gall to [forbid some kind of links](https://info.orcid.org/brand-guidelines/#h-orcid-logos-and-icons) to their websites:

> We do not currently allow references to authorized ORCID iDs without using the iD icon.

So a link like “[Damien Goutte-Gattat](https://orcid.org/0000-0002-6095-8718)” is _not allowed_, since the ORCID iD icon is missing. Instead it should be something like

* [![](https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png) https://orcid.org/0000-0002-6095-8718](https://orcid.org/0000-0002-6095-8718) (icon + full URL)
* [![](https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png) 0000-0002-6095-8718](https://orcid.org/0000-0002-6095-8718) (icon + ORCID iD)
* [Damien Goutte-Gattat ![](https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png)](https://orcid.org/0000-0002-6095-8718) (name + icon)

(It will never not amaze me that some people believe they can dictate whether and how other people can create links… But I am more used to see that from big old-fashioned business brands that know nothing about how the web works, it’s disappointing to see that from an organisation like ORCID that is both not-for-profit and supposedly tech-savvy. Oh well…)

Your call whether you want to follow their rules or not. Personally, I am inclined to disregard their rules entirely, and to create links on our own website in whatever way we want, whether they _allow_ it or not.

Reply:
Shall we leave updating this list to a separate issue? If so, you could turn your comment into an issue and assign to me, I will update the contributor lists and think about the ID icon.

Reply:
Fine with me.

> I will update the contributor lists.

If we agree that, in the docs/spec, the list of contributors is understood as the list of “all the persons who have ever contributed, regardless of whether they are still contributors _now_”, then updating the list should only be a matter of _adding_ the persons who started contributing after that list was created, right? Basically the list is append-only – no one should ever be removed from it. Do we agree on that?

Reply:
You are right, which means that these list should, indeed, be managed separately. I just need to do a 1 time sync between the github core sssom developer list now, and then these list are forever disconnected, because, when someone chooses to leave the sssom-core team for whatever reason, this does not mean they loose editor status